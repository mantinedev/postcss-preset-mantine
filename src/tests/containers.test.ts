import { testTransform } from './utils';

const rem = `
.card {
  @container (min-width: rem(32px)) {
    background: red;
  }
}
`;

const em = `
.card {
  @container (min-width: em(32px)) {
    background: red;
  }
}
`;

describe('@container', () => {
  it('transforms rem', async () => {
    const res = await testTransform(rem);
    expect(res.css).toMatchSnapshot();
  });

  it('transforms em', async () => {
    const res = await testTransform(em);
    expect(res.css).toMatchSnapshot();
  });
});
