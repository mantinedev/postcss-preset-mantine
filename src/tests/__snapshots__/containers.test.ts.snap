// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@container transforms em 1`] = `
"
@container (min-width: 2em) {
.card {
    background: red
}
  }
"
`;

exports[`@container transforms rem 1`] = `
"
@container (min-width: 2rem) {
.card {
    background: red
}
  }
"
`;
